omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2Test.java

```diff
@@ -23,7 +23,7 @@ public Person newObject() {
 	
 	@Override
 	public String getDisplayProperty() {
-		return "Mr. Horatio Test Hornblower";
```

Comment:
What led to this change?

Reply:
That's a good question :slightly_smiling_face: I think that including  `legacyui-omod` changes the test data for unit tests and the test above started to fail. However, I wasn't able to find what piece of code in `legacyui-omod` alters this person's data. I was hoping that you will be able to help me figure out what's going on here.